scripts/data/getCountryPolygon.R

```diff
@@ -35,14 +33,18 @@ country_name <- countrycode(
 
 biab_output("country", country_name)
 
+
 if (is.null(input$region)) { # pull study area polygon from rnaturalearth
   # pull whole country
-  print("pulling country polygon")
-  country_polygon <- ne_countries(country = country_name, type = "countries", scale = 10)
+  url <- paste0("https://github.com/wmgeolab/geoBoundaries/raw/9469f09/releaseData/gbOpen/", input$country, "/ADM0/geoBoundaries-", input$country,"-ADM0.geojson")
```

Comment:
Be aware that this will always take the data from commit 9469f09 and will not be updated if the repo has new updates (last commit was 2 years ago though, official borders don't change that very often) If this is intended, fine. If not, you could use `raw/refs/heads/main` instead as below

https://github.com/wmgeolab/geoBoundaries/raw/refs/heads/main/releaseData/gbOpen/ALB/ADM0/geoBoundaries-ALB-ADM0.geojson